public_html/pixels2.css

```diff
@@ -0,0 +1,127 @@
+
+* {
+    /* select all elements */
+    box-sizing: border-box;
+}
+
+.rounded-box {
+    /* reusable rounded box for content within the site-container */
+    background-color: rgb(241,241,241);
+    border-radius: 20px;
+    padding: 28px;
+    margin-bottom: 20px;
+}
+
+body {
+    /* body of site */
+    background-color: white;
+    padding: 23px;
+    font-family: Arial;
+    margin: 0px;
+}
+
+.site-container {
+    /* Container for whole site */
+    width: 1040px;
+    margin: 0 auto;
+    padding: 10px;
+}
+
+.site-header {
+    /* Container for site header */
+    height: 105px;
+    padding: 18px;
+    overflow: auto;
+}
+
+.site-header__logo {
+    /* Site Header logo */
+    float: left;
+    width: 200px;
+    height: 66px;
+    background-image: url('https://cloud.githubusercontent.com/assets/1764468/16341829/0857e36e-3a27-11e6-95ce-b2136c0fadb1.png');
+    background-repeat: no-repeat;
+}
+
+.navbar-menu {
+    /* Site navigation menu */
+    float: right;
+}
+
+.navbar-menu__item {
+    /* Each menu item */
+    line-height:67px;
+    padding: 20px;
+    font-size: 18px;
+    font-weight: bold;
+    color: black;
+    text-decoration: none;
+}
+
+.navbar-menu__item:hover {
+    /* Hover over menu item */
+    text-decoration: underline;
+}
+
+.main-article {
+    /* Container for the main article of the site */
+    overflow: auto;
+    vertical-align: top;
+    min-height: 415px;
+    padding-right: 35px;
+    position: relative;
+}
+
+.main-article__image {
+    /* main image in the main article */
+    float: left;    
+    vertical-align: top;
+    padding-right: 30px; /*padding here instead of content for better alignment at small screen widths */
```

Comment:
Not sure this comment makes sense with the current implementation. Maybe just remove?